src/player/sdl.c

```diff
@@ -57,17 +57,17 @@ int main(int argc, char **argv)
 			SDL_Init(SDL_INIT_VIDEO | SDL_INIT_AUDIO);
 
 			{
-				SDL_Window* window = SDL_CreateWindow("TIC-80 SDL demo", SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED, TIC80_FULLWIDTH, TIC80_FULLHEIGHT, SDL_WINDOW_SHOWN | SDL_WINDOW_RESIZABLE);
+				SDL_Window* window = SDL_CreateWindow("TIC-80 SDL demo", SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED, TIC80_FULLWIDTH * 3, TIC80_FULLHEIGHT * 3, SDL_WINDOW_SHOWN | SDL_WINDOW_RESIZABLE);
```

Comment:
Could you pls use a constant or compiler definition instead `3` here?
Thank you :)